test/torchaudio_unittest/transducer_test.py

```diff
@@ -241,6 +242,14 @@ def compute_with_pytorch_transducer(data):
     return costs, gradients
 
 
+def skipIfNoRNNT(test_item):
+    try:
+        import torch.ops.torchaudio.rnnt_loss as _
```

Comment:
Import looks a bit strange, can you just access the attribute?

```suggestion
        _ = torch.ops.torchaudio.rnn_loss
```

Reply:
Thanks! I was trying to suppress the style check error for simple `import torch.ops.torchaudio.rnnt_loss`